lib/content/audio/approaching.ex

```diff
@@ -54,6 +54,8 @@ defmodule Content.Audio.Approaching do
     @with_all_new_red_line_cars "893"
     # audio: "It is a shorter 4-car train. Move toward the front of the train to board, and stand back from the platform edge.", visual: "Please move to front of the train to board."
     @four_car_train_message "922"
+    # "Please stand back from the platform edge."
+    @stand_back_message "925"
```

Comment:
I take it (heh) we sent this to ARINC as a new take, rather than telling them to overwrite the take we previously expected to use, 898 "please stand back from the yellow line"?